src/stream.c

```diff
@@ -1518,6 +1526,38 @@ const char *stream_name(const struct stream *strm)
 }
 
 
+/**
+ * Get the value of the RTCP Canonical end-point identifier
+ *
+ * @param strm Stream object
+ *
+ * @return Canonical end-point identifier
+ */
+const char *stream_cname(const struct stream *strm)
+{
+	if (!strm)
+		return NULL;
+
+	return strm->cname;
+}
+
+
+/**
+ * Get the value of the RTCP Canonical end-point identifier
```

Comment:
here also!

Reply:
@cspiel1 , same question...I'm not entirely clear about the request - should I change the contents of the comments to reflect something specific or run a doxygen command to generate update documentation?